mock_mongosqltranslate/rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "nightly"
```

Comment:
I might have missed it while I was out, but what do we need nightly for?

Reply:
I added that for the mock library when I was more closely following what was in the design with this line:
```
// Convert from bson::Document to an array of BSON bytes
let bytes = bson::to_vec(&result_docs).expect(<message>);
let (ptr, len, cap) = bytes.into_raw_parts();  <-----
(ptr.cast_const(), len, cap)
```
The [Vec::into_raw_parts()](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.into_raw_parts) needed the nightly toolchain.  I modified the library to not need that function and forgot to remove this file, I'll do that now it should work without it.  Thanks for bringing it up.